src/app/models/Deposits.ts

```diff
@@ -0,0 +1,24 @@
+export interface DepositsResponse {
```

Comment:
Please name this file as a single class entity, Also let it contain just properties of that typical class.
Use some of the rxjs operators to customize conditionally what components take in from the backend.

Reply:
Hey @CryceTruly, l have worked on this thanks!